cmd/flowrunner/runner_test.go

```diff
@@ -21,23 +20,22 @@ import (
 )
 
 var flowTests = []struct {
-	assets  string
-	contact string
-	output  string
+	assets string
+	output string
```

Comment:
'output' contains a trigger and the events, so since contact is now on trigger, these tests become a bit more self-contained. Previously contact and environment were being set programatically so not appearing anywhere in the output